### DISCUSSION TITLE
Line 6 Floorboard

### DISCUSSION THREAD

**Discussion by jimhiggs** [alf45tar/PedalinoMini#407] [Show and tell]
Had this sitting around and could get no interested buyers so, initially I considered breaking it for parts... Then I decided on this.
Using my trust grinder I cut off the right side (exp pedals - made it too big for me). Cut the middle out of the base so ends and screw holes lined up. Leds replace with WS2812. Little bit of rewiring of the three switch pcbs, ditched the line 6 control boards and here we go.
Larger 1.3" Oled. 
3 x 6 switch ladders, 2 external sockets for exp pedals or extra switches.
Rotary encoder next to display for bank select and push switch acts as switch 7 (boot switch).
Usb midi o/p on back - also allows reprogramming of pro micro
Usb input on side for reprogramming esp32.
![IMG_20221208_205055047](https://user-images.githubusercontent.com/33665935/206598010-81ac7ab8-e267-41f5-8c59-5f51bd0de007.jpg)

Extra hole on back for 3.5mm stereo socket for conventional Midi Out. (My irig pro came with 5 pin to 3pin cable adaptors for midi in/out).
![IMG_20221208_184047764](https://user-images.githubusercontent.com/33665935/206585706-afa067fa-f5a5-4c92-a734-cba0470ac951.jpg)
![IMG_20221208_184028347](https://user-images.githubusercontent.com/33665935/206585745-7fb75ef3-570c-49a4-82c0-4ed2717a110b.jpg)
![IMG_20221208_184241654](https://user-images.githubusercontent.com/33665935/206585810-24f468e4-3c19-4874-ae42-9fe6d10b43dc.jpg)
![IMG_20221208_184301205](https://user-images.githubusercontent.com/33665935/206585840-e51c9633-0203-415d-bfea-ab97160d9d4c.jpg)

![IMG_20221208_184318960](https://user-images.githubusercontent.com/33665935/206585823-478e2b9f-f14a-45ea-8c69-80beecd9b570.jpg)

![IMG_20230202_213945546](https://user-images.githubusercontent.com/33665935/216710932-f6424d17-3362-4468-8f79-1179b1c2e28e.jpg)

**Comment by Celticpure** [comment]
This is very cool

**Comment by rigr** [comment]
Please allow a look at the guts (inside - when you open it up the next time). Looks great!

**Reply by jimhiggs** [reply]
Maybe I am embarrassed about my wiring neatness.

**Reply by rigr** [reply]
:) ok. Mine even looks bad from the outside, but as long as it works..

**Reply by jimhiggs** [reply]
I will open it up and get some pics.

**Reply by rigr** [reply]
It might encourage others to build - from outside yours looks great. :)

**Reply by jimhiggs** [reply]
Kudos goes to line 6 I guess.

**Reply by jimhiggs** [reply]
Inside pic added, well half of it anyway.

**Reply by rigr** [reply]
Looks ok, I think. :)

**Comment by Wapata** [comment]
It looks wonderful !
Is the wireless working great with the metal case ?

**Reply by jimhiggs** [reply]
I would not say wonderful. It works most of the time but occasionally has difficulty connecting. I might change to the esp32 with external antenna to see how that works.

**Reply by dimarzio49** [reply]
How did you manage to use so many ws2812 LEDs, I only have 10?

**Reply by jimhiggs** [reply]
What do you believe is the limiting factor? I would have thought hundred or more would be doable.
